services/app-web/src/components/FileUpload/FileProcessor/FileProcessor.tsx

```diff
@@ -97,32 +82,19 @@ type FileProcessorProps = {
     item: FileItemT
     deleteItem: (item: FileItemT) => void
     retryItem: (item: FileItemT) => void
-    renderMode: 'table' | 'list'
-    handleCheckboxClick: (event: React.ChangeEvent<HTMLInputElement>) => void
-    isContractOnly?: boolean
-    shouldValidate?: boolean
 }
 export const FileProcessor = ({
     item,
     deleteItem,
     retryItem,
-    renderMode,
-    handleCheckboxClick,
-    isContractOnly,
-    shouldValidate,
 }: FileProcessorProps): React.ReactElement => {
     const { name, status, file } = item
-    const isRateSupporting = item.documentCategories.includes('RATES_RELATED')
-    const isContractSupporting =
-        item.documentCategories.includes('CONTRACT_RELATED')
```

Comment:
This is good. Thank you for looking carefully at this file.  I was pretty frustrated when we had to add this because we were leaking all this domain data into what was supposed to be a multipurpose re-useable components (e.g. it could be user avatar image upload, it could be any random upload). Now removing this we are back to a truly re-useable domain agnostic component